docs/react-testing-library/example-intro.md

```diff
@@ -122,12 +122,12 @@ export default function Fetch({ url }) {
   const buttonText = buttonClicked ? 'Ok' : 'Load Greeting'
 
   return (
-    <div>
+    <>
       <button onClick={fetchGreeting} disabled={buttonClicked}>
         {buttonText}
       </button>
-      {greeting ? <h1>{greeting}</h1> : null}
-    </div>
+      {greeting && <h1>{greeting}</h1>}
```

Comment:
Similar reason here. Ternaries are perfectly fine.

Reply:
I actually prefer ternaries so that you don't actually end up with, say `<div>0</div>` when the value is falsy but not actually `null` or `false`.